extractor.py

```diff
@@ -35,8 +36,14 @@ def fetch_codes(browser):
 else:
     mailbox = IMAP4(host=config.IMAP_HOST, port=config.IMAP_PORT)
 
+#Prompt for password if not specified
+if config.IMAP_PASSWORD:
+    mypass = config.IMAP_PASSWORD
+else:
+    mypass = getpass.getpass()
+
```

Comment:
Instead of doing it this way, I would suggest the following:

```python
imap_pass = config.IMAP_PASSWORD
if not imap_pass:
    imap_pass = getpass.getpass()
```